ts/screens/main/MessagesScreen.tsx

```diff
@@ -1,24 +1,82 @@
-import { Container } from "native-base";
+import {
+  Body,
+  Container,
+  Content,
+  H1,
+  List,
+  Tab,
+  Tabs,
+  Text,
+  View
+} from "native-base";
 import * as React from "react";
-import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { connect } from "react-redux";
+import { CreatedMessageWithContent } from "../../../definitions/backend/CreatedMessageWithContent";
+import { ReduxProps } from "../../actions/types";
+import messages from "../../api/mock/messages.json";
+import MessageComponent from "../../components/MessageComponent";
+import AppHeader from "../../components/ui/AppHeader";
+import I18n from "../../i18n";
+import variables from "../../theme/variables";
 
-export type OwnProps = {
-  navigation: NavigationScreenProp<NavigationState>;
+type ReduxMappedProps = {};
+export type OwnProps = {};
+type Props = ReduxMappedProps & ReduxProps & OwnProps;
+type State = {};
+type ListOfCreatedMessageWithContentInterface = {
+  [id: string]: CreatedMessageWithContent;
 };
 
-export type Props = OwnProps;
-
 /**
  * This screen show the messages to the authenticated user.
  */
-class MessagesScreen extends React.Component<Props, never> {
-  constructor(props: Props) {
-    super(props);
-  }
+class MessagesScreen extends React.Component<Props, State> {
+  public renderMessageList = (
+    list: ListOfCreatedMessageWithContentInterface
+  ) => {
+    return Object.keys(list).map(key => {
+      return (
+        <MessageComponent
+          key={key}
+          sender={list[key].sender_service_id}
+          subject={list[key].content.subject}
+          date={list[key].created_at}
+        />
+      );
+    });
+  };
 
   public render() {
-    return <Container />;
+    const messagesById: ListOfCreatedMessageWithContentInterface =
+      messages.byId;
+    return (
+      <Container>
+        <AppHeader>
+          <Body>
+            <Text>{I18n.t("messages.headerTitle")}</Text>
+          </Body>
+        </AppHeader>
+        <Content>
+          <View spacer={true} />
+          <H1>{I18n.t("messages.contentTitle")}</H1>
+          <Tabs
+            tabBarUnderlineStyle={{
+              backgroundColor: variables.brandDarkenBlue
+            }}
+            initialPage={0}
+          >
+            <Tab heading={I18n.t("tabMessages.itemAll")}>
+              <View spacer={true} large={true} />
+              <List>{this.renderMessageList(messagesById)}</List>
```

Comment:
NativeBase List is going to be deprecated soon. Internally it uses [ListView](https://facebook.github.io/react-native/docs/listview.html) that is DEPRECATED. Use a [FlatList](https://facebook.github.io/react-native/docs/flatlist.html).